arch/X86/X86Mapping.c

```diff
@@ -1819,6 +1819,27 @@ static bool valid_repe(cs_struct *h, unsigned int opcode)
 	return false;
 }
 
+// given MCInst's id, find out if this insn is valid for NOTRACK prefix
+// NOTRACK prefix is valid for CALL/JMP
+static bool valid_notrack(cs_struct *h, unsigned int opcode)
+{
+	unsigned int id;
+	unsigned int i = find_insn(opcode);
+	if (i != -1) {
+		id = insns[i].mapid;
+		switch(id) {
+			default:
+				return false;
+			case X86_INS_CALL:
+			case X86_INS_JMP:
+                return true;
```

Comment:
This line has spaces, but everything else has tabs.

Reply:
I just submitted a PR to the main repo: https://github.com/capstone-engine/capstone/pull/2002.

Also, added some tests in issues.cs.